tests/routes/datasets/test_versions.py

```diff
@@ -180,8 +183,8 @@ async def test_version_metadata(async_client: AsyncClient):
     assert response.status_code == 201
 
     assert (
-        response.json()["data"]["metadata"]["resolution"]
-        == version_metadata["resolution"]
+        response.json()["data"]["metadata"]["spatial_resolution"]
+        == version_metadata["spatial_resolution"]
```

Comment:
Since you fix this case in the deprecated test, maybe also fix the case at line 212 (change resolution to "spatial_resolution", and also probably "title" to "spatial_resolution".